README.md

```diff
@@ -2,8 +2,14 @@
 
 ## Installation
 
+For webpack v2 users:
+
 `npm install --save json-loader`
 
+For webpack v1 users:
+
+`npm install --save json-loader@0.5.4`

```

Comment:
Yes, that would in theory work. But it would be much easier to just use `0.5.4`.